docs/source/_static/install_dependencies.sh

```diff
@@ -114,18 +113,15 @@ elif [ -f /etc/os-release ]; then
             sudo apt-get install -y "${ubuntu_arm_pkgs[@]}"
             python3 -m pip install --upgrade pip
         fi
-        ;;
-    fedora)
+    elif [[ "$ID" == fedora ]]; then
```

Comment:
Minor thing, I'd wrap this variable in`"` to keep consistent with the rest of the checks